package.json

```diff
@@ -38,6 +36,7 @@
     "@bedrock/core": "^6.0.0",
     "@bedrock/express": "^8.0.0",
     "@bedrock/passport": "^11.0.0",
+    "@bedrock/turnstile": "^1.0.1",
```

Comment:
test is currently failing locally `@bedrock/turnstile` needs to be added to the test deps.